src/Akeneo/Connectivity/Connection/back/tests/EndToEnd/Webhook/UpdateEventSubscriptionEndToEnd.php

```diff
@@ -44,7 +44,7 @@ public function test_it_updates_an_event_subscription(): void
         $data = [
             'code' => $connection->code(),
             'enabled' => true,
-            'url' => 'http://localhost',
+            'url' => 'http://example.com',
```

Comment:
```suggestion
            'url' => 'http://example.test',
```
It's minor, but if you can, use the `.test` domain extension that was created/reserved for this purpose (and you will never risk to call a real domain)

Reply:
In this case, if `FakeDnsLookup.php` were to be removed the test will probably pass, and call `example.com` for real.

Reply:
Fine to me, anyway, I was not resolving the DNS in the tests, it's mocked, even for e2e.